torchquad/integration/vegas_map.py

```diff
@@ -7,16 +7,16 @@
 
 
 class VEGASMap:
-    """The map used for VEGAS Enhanced. Refer to https://arxiv.org/abs/2009.05112.
+    """The map used for VEGAS Enhanced. Refer to https://arxiv.org/abs/2009.05112 .
     Implementation is inspired by https://github.com/ycwu1030/CIGAR/ .
-    EQ refers to equation in the paper.
+    EQ <n> refers to equation <n> in the above paper.
     """
 
     def __init__(self, dim, integration_domain, N_intervals=100, alpha=0.5) -> None:
-        """Initializes VEGAS' Enhanced adaptive map
+        """Initializes VEGAS Enhanced' adaptive map
```

Comment:
It is a bit of a tongue twister, but I still think `VEGAS Enhanced's` is correct?